text/2019-11-21-key-visualizer-zh_CN.md

```diff
@@ -0,0 +1,243 @@
+# RFC: `KeyVisualizer` 可视化热点图
```

Comment:
At this time it's not a convention to include Chinese versions of RFCs, maybe you can make a GIST and link to it in the English one?